packages/components/ng-ovh-payment-method/src/legacy/mean/payment-mean-type.class.js

```diff
@@ -1,15 +1,15 @@
 import { snakeCase } from 'lodash-es';
 
-import OvhPaymentMethodType from '../../payment-method-type.class';
+import OvhAvailablePaymentMethod from '../../available-payment-method.class';
 
 export default class OvhPaymentMeanType {
   constructor(options = {}) {
     this.meanType = options.value;
     this.registerable = options.registerable;
   }
 
-  toPaymentMethodType() {
-    return new OvhPaymentMethodType({
+  toAvailablePaymentMethod() {
```

Comment:
I didn't see any `BREAKING CHANGE` comment but this change looks like one as package is published

Reply:
set `toPaymentMethodType` to deprecated to avoid breaking change.